packages/image-cloze-association/src/root.jsx

```diff
@@ -33,7 +33,7 @@ const styles = (theme) => ({
   },
 });
 
-class ImageClozeAssociationComponent extends React.Component {
+export class ImageClozeAssociationComponent extends React.Component {
```

Comment:
Don't we already export the component at the bottom of the file?

Reply:
for testing purposes, I needed direct access to the unwrapped class component —this change is only for enabling better unit testing.